HW03 Maria Evers.Rmd

```diff
@@ -0,0 +1,128 @@
+  
+#Making the graph look nice 
+  
+  theme(plot.title = element_text(hjust = 0.5, size = 14), axis.text = element_text(size = 12, color = "black"), axis.title = element_text(size = 14), panel.grid.major = element_blank(), panel.grid.minor = element_blank(), panel.background = element_blank(), axis.line.x.bottom = element_line(color = "black"), axis.line.y.left = element_line(color = "black")) +
+  
+#Adding stars for significance (pvalue calculated in Excel). I added each manually, but I am sure there are other ways to add these using pvalues as a column in your data. 
+  
+  annotate("text", label = "*" , x = 4, y = 150, color = "black", size = 7) +
+  annotate("text", label = "*" , x = 5, y = 150, color = "black", size = 7) +
+  annotate("text", label = "***" , x = 6, y = 175, color = "black", size = 7) +
+  annotate("text", label = "**" , x = 7, y = 175, color = "black", size = 7) +
+  annotate("text", label = "**" , x = 8, y = 185, color = "black", size = 7) +
+  annotate("text", label = "***" , x = 9, y = 210, color = "black", size = 7) +
+  annotate("text", label = "***" , x = 10, y = 275, color = "black", size = 7) 
+
+
+
+```
+
+### Next, I am graphing some RNASeq Data that we would like to highlight in an upcoming publication. 
+### This data is looking at changes in A549 NSCLC cells RNA expression levels in specific EMT-related genes after treatment with ATP or TGF-B for 2 or 6 hours.
+### I think this plot needs work, but I need to produce multi-panel figures like this with RNASeq data and Metabolomics data for a publication I'm working on, so **any input on this graph specifically is greatly appreciated!**
+
+```{r}
+library(readxl)
+
+#Loading in the Excel file. I went ahead and changed my data from the "wide" format to the "long" format in excel before importing. Instead of each treatment being its own column, treatment is now a column itself.
+
+RNASeqEMT <- read_excel("~/Documents/RNA seq analysis/Diff Exp Genes accumulated data.xlsx", 
+    sheet = "R data")
+    
+#Previewing structure of the data set 
+
+str(RNASeqEMT)
+
+#Graphing the dataset into a multi-panel bar chart, unfortunately I could not reorder my treatments to the desired order (control first) 
+
+ggplot(RNASeqEMT, aes(Gene_name, log2FC, color = Treatment, fill = Treatment)) + 
+  geom_bar(aes(x = Treatment), stat = 'identity') +
+  facet_wrap(~ Gene_name) +
+  
+#Adding labels 
+
+  labs(title = "ATP and TGF-B treatment alter the transcription of several EMT \n or mesenchymal-type genes", x = element_blank(), y = "log2(FC) values") +
+  
+#Making the plot look nice
+
+  scale_color_brewer(palette = "Paired") +
+  
+  scale_fill_brewer(palette = "Paired") +
+  
+  theme_bw() +
+  
+  theme(plot.title = element_text(hjust = 0.5, size = 14), axis.text = element_text(size = 8, color = "black"), axis.title = element_text(size = 14), axis.text.x = element_blank(), axis.ticks.x = element_blank()) 
+  
```

Comment:
This plot looks good to me! Only one thing, if this is for publication, on the legend, it might be better to have "2h_ATP_treatment" than "2_ATP", so includes more info. Also, might be good to remove the background grid, however, this is very personal taste, which can be nonsense, lol.